packages/protocol/test/governance/governance.ts

```diff
@@ -75,6 +75,15 @@ interface Transaction {
 // hard coded in ganache
 const EPOCH = 100
 
+async function mineToNextEpoch(web3) {
+  const bn = web3.eth.getBlockNumber()
+  if (bn % EPOCH < 50) {
+    await mineBlocks(EPOCH + 20, web3)
+  } else {
+    await mineBlocks(EPOCH - 20, web3)
+  }
```

Comment:
What's the rationale behind the +20 / -20?

Reply:
Well +20 is probably not useful, but -20 makes sure that the epoch won't change twice. Perhaps still need different handling for the case `bn % EPOCH == 0`.